.github/scripts/run_tests.sh

```diff
@@ -1,34 +1,32 @@
 set -ex
 
 function run_torch_xla_python_tests() {
-  PYTORCH_DIR=$1
-  XLA_DIR=$2
-  USE_COVERAGE="${3:-0}"
+  XLA_DIR=$1
```

Comment:
What's the reason for removing the `PYTORCH_DIR` argument?

Reply:
I am wondering why we need PYTORCH_DIR in those tests. The PYTORCH_DIR passed in to those bash scripts are not absolute path. It's just something like `pytorch/`.

Reply:
And we didn't use PYTORCH_DIR variable in the function. So I cleaned them up.

Reply:
I thought that variable was being used by one of the downstream scripts. If the tests still pass, this is fine though.